src/Components/UI/input/MyInput.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const MyInput = (props) => {
```

Comment:
Each component that has props, has to have proptypes specified 
 
But actually doesn't look like there is any sence of having this component as separate, as it does nothing (neither in styles or logic), just passes props as they are to other component